### DISCUSSION TITLE
Approval

### DISCUSSION THREAD

**Discussion by DeadlyShroud** [ROM-builders/temporary#23693] [General]
Hi, can you please approve me?
Here's all my stuff:
[GitHub](https://github.com/DeadlyShroud) 
[GitLab](https://gitlab.com/DeadlyShroud) 
[Successful Build](https://sourceforge.net/projects/rom-builders/files/munch/AlphaDroid-13.0-20230826-munch-vanilla-v1.8.zip/download) 
[Distribution](https://t.me/PocoF4updates/765) 
Thanks!

**Comment by Apon77** [comment]
Seems like fake account
Also follow #learn instruction from telegram group
